.github/workflows/build_and_test.yml

```diff
@@ -8,15 +8,18 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
This should stay 20.04 to make sure it's not updated to 20.10

Reply:
Done.

Reply:
Thanks